src/markdown-pages/install-with-kurl/upgrading.md

```diff
@@ -18,7 +18,27 @@ After upgrading all primaries the same operation will be performed sequentially
 It is possible to upgrade from any Kubernetes minor versions to the latest supported Kubernetes version using a single spec.
 This upgrade process will step through minor versions one at a time.
 For example, upgrades from Kubernetes 1.19.x to 1.26.x will step through versions 1.20.x, 1.21x, 1.22.x, 1.23.x, 1.24.x, and 1.25.x before installing 1.26.x.
-Upgrades without internet access may prompt the end-user to download supplemental packages.
+
+To upgrade Kubernetes by more than one minor version in air gapped instances without internet access, the end-user must provide a supplemental package that includes the assets required for the upgrade. The script prompts the user to provide the required package during upgrade. For example:
+
+```bash
+⚙  Upgrading Kubernetes from 1.23.17 to 1.26.3
+This involves upgrading from 1.23 to 1.24, 1.24 to 1.25, and 1.25 to 1.26.
+This may take some time.
+⚙  Downloading assets required for Kubernetes 1.23.17 to 1.26.3 upgrade
+The following packages are not available locally, and are required:
+    kubernetes-1.24.12.tar.gz
+    kubernetes-1.25.8.tar.gz
+
+You can download them with the following command:
+
+    curl -LO https://kurl.sh/bundle/version/v2023.04.24-0/19d41b7/packages/kubernetes-1.24.12,kubernetes-1.25.8.tar.gz
```

Comment:
```suggestion
    curl -LO https://kurl.sh/bundle/version/v2023.04.24-0/19d41b7/packages/{kubernetes-1.24.12,kubernetes-1.25.8.tar.gz}
```

Reply:
this is actual output from the install script.

Reply:
oh, right! Nevertheless, I still think we have to update this message since executing `curl -LO https://kurl.sh/bundle/version/v2023.04.24-0/19d41b7/packages/kubernetes-1.24.12,kubernetes-1.25.8.tar.gz` will download an archive with the name `kubernetes-1.24.12,kubernetes-1.25.8.tar.gz`.